codespell_lib/data/dictionary.txt

```diff
@@ -8977,6 +8980,7 @@ degress->degrees, digress,
 deimiter->delimiter
 deine->define
 deined->denied, defined,
+deines->denies, defines, defined,
```

Comment:
I don't think the last one is very likely here, unless you've seen it in the wild?
```suggestion
deines->denies, defines,
```

Reply:
Yes, I've seen it. Not sure where. But the key proximity of the `s` and `d` are to blame i would guess

Reply:
Fair enough.